full_bubble_model.py

```diff
@@ -15,20 +19,104 @@
 try:
     import parameters as par   # numeric constants and coefficents
     importlib.reload(par)   # reload changes you made
-    print(f'model: {par.model}')
 except:
-    print('Error, \'parameters.py\' not found')
+    print(print(colored('Error, \'parameters.py\' not found','red')))
+try:
+    import excitation
+    importlib.reload(excitation)
+except:
+    try:
+        import Bubble_dynamics_simulation.excitation as excitation
+        importlib.reload(excitation)
+    except:
+        print(colored(f'Error, \'excitation.py\' not found', 'red'))    
+
+"""________________________________Settings________________________________"""
+
+enable_heat_transfer = True
```

Comment:
These settings are very important. You shouldn't modify them trough code in other files, only here. Always check them, before you start something. The used reaction model and these settings will be displayed once you import this file